### DISCUSSION TITLE
No module named

### DISCUSSION THREAD

**Discussion by tekishmain** [opengeos/leafmap#790] [Q&A] [answered]
I installed leafmap but couldn't be imported on jupyter notebook

### Environment Information

-   leaf map version: the latest one
-   Python version: 3.10
-   Operating System: Windows

### Description

I installed using pip install leafmap but not be imported

### What I Did
import os
import time
import requests
import leafmap.maplibregl as leafmap
```
I found this,
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-fff1bff737f4> in <module>
      2 import time
      3 import requests
----> 4 import leafmap.maplibregl as leafmap

ModuleNotFoundError: No module named 'leafmap'

what is the problem?

Thanks

```

**Comment by giswqs** [comment, marked as answer]
This is very likely your own computer problem. Try creating a fresh conda env to install leafmap.

```bash
conda create -n geo python=3.11
conda activate geo
conda install -c conda-forge leafmap maplibre
pip install "leafmap[maplibre]"
jupyter lab
```
Alternatively, use Google Colab. 
https://colab.research.google.com/github/opengeos/leafmap/blob/master/docs/maplibre/3d_buildings.ipynb
